generate.go

```diff
@@ -6,10 +6,4 @@ package main
 //go:generate go run sigs.k8s.io/controller-tools/cmd/controller-gen crd:trivialVersions=true rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=${CRD_ROOT_DIR}/bases crd:crdVersions=v1
 
 // Generate API reference documentation
-////go:generate go run github.com/elastic/crd-ref-docs --source-path=api/v1alpha1 --config=docs/api-gen-config.yaml --renderer=asciidoctor --templates-dir=docs/api-templates --output-path=${CRD_DOCS_REF_PATH}
-
-import (
-	_ "github.com/elastic/crd-ref-docs"
-	_ "sigs.k8s.io/controller-tools/cmd/controller-gen"
-	_ "sigs.k8s.io/kustomize/kustomize/v3"
-)
```

Comment:
Are you really sure this isn't needed when running `make generate`?
(note it might not output anything if the generation failed)

Reply:
yes. they are already covered in `tools.go`.

Reply:
besides, they're not part of the same file in k8up